confignode/src/main/java/org/apache/iotdb/confignode/physical/PhysicalPlanType.java

```diff
@@ -24,11 +24,13 @@
   SetStorageGroup,
   DeleteStorageGroup,
   QueryStorageGroupSchema,
-  CreateRegion,
-  QueryDataPartition,
-  ApplyDataPartition,
-  QuerySchemaPartition,
-  ApplySchemaPartition,
+  CreateRegions,
+  GetDataPartition,
+  CreateDataPartition,
+  GetOrCreateDataPartition,
+  GetSchemaPartition,
+  CreateSchemaPartition,
+  GetOrCreateSchemaPartition,
```

Comment:
Are these `getOrCreate` plans judged below the consensus layer? And if so, how do we guarantee certainty?

Reply:
No. Get or create means, we will check the exsistence of each Partition upward the consensus layer. If not, we will generate a certain CreatePlan and write through consensus layer. Finally we get the result.

Reply:
I see